python/dgl/nn/pytorch/conv/dotgatconv.py

```diff
@@ -3,12 +3,13 @@
 from torch import nn
 
 from .... import function as fn
-from ....ops import edge_softmax
+from ..softmax import edge_softmax
 from ....utils import expand_as_pair
 
 
 class DotGatConv(nn.Module):
```

Comment:
btw, I was wondering where do we need this module? I think it's exactly the same as Transformer which we have already supported.